app/src/services/dataset.service.js

```diff
@@ -156,6 +183,9 @@ class DatasetService {
                 } else if (query[param] === 'false') {
                     query.subscribable = { $in: [null, false, {}] };
                 }
+            } else if (DatasetService.isApplicationConfigFilter(param)) {
```

Comment:
maybe use https://lodash.com/docs/4.17.15#get instead?

Reply:
Actaully, what I was looking for here is lodash.set, but thanks for the tip!